### PR TITLE
Add new cmdlet for retrieving policy aliases

### PR DESCRIPTION
**Pull request by mentat9**
## Description

New cmdlet for retrieving policy aliases. Doesn't depend on swagger changes.

## Checklist

- [X] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [X] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [X] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by maddieclayton** [comment]
Please either link your cmdlet review or fill our a new one here: https://github.com/Azure/azure-powershell-cmdlet-review-pr

**Comment by mentat9** [comment]
Link to new cmdlet review issue:

https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/148

File: src/ResourceManager/Resources/Commands.Resources/help/Get-AzureRmPolicyAliases.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
+---
+external help file: Microsoft.Azure.Commands.ResourceManager.Cmdlets.dll-Help.xml
+Module Name: AzureRM.Resources
+online version:
```
Comment (cormacpayne) [inline src/ResourceManager/Resources/Commands.Resources/help/Get-AzureRmPolicyAliases.md]: @mentat9 from the `ValidateHelpIssues.csv` file from the build:

> No entry was found for the online version field in the header. The corresponding URL should be the following: https://docs.microsoft.com/en-us/powershell/module/azurerm.resources/get-azurermpolicyaliases

File: src/ResourceManager/Resources/Commands.Resources/help/Get-AzureRmPolicyAliases.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+schema: 2.0.0
+---
+
+# Get-AzureRmPolicyAliases
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
```
Comment (cormacpayne) [inline src/ResourceManager/Resources/Commands.Resources/help/Get-AzureRmPolicyAliases.md]: @mentat9 please make sure there is content in the template sections provided (synopsis, description, examples)

**Comment by mentat9** [comment]
Made the revisions to address feedback: @cormacpayne please take a look and close out, thanks.

**Comment by mentat9** [comment]
Changes done: rename cmdlet, add output to examples.

**Comment by cormacpayne** [comment]
@mentat9 tests are failing that use the `AzureRM.Resources` module -- it seems there's an issue when importing the module, but the tests are giving the bubbled up error message. Would you mind trying to import the local module and see what the error is?

**Comment by mentat9** [comment]
Loading the module locally is working fine and has been for weeks. I'll see if I can figure out what's wrong with the tests.

**Comment by mentat9** [comment]
Ah, looks like I forgot to add the generated format.ps1xml file to the project. It was working locally because I had a copy in the output folder already. I just pushed that change now. Hopefully that will clean up the tests.

**Comment by mentat9** [comment]
Thanks for your help, **Powershell** team!

**Pull request merged**